sympy/physics/hep/tests/test_gamma_matrices.py

```diff
@@ -257,10 +257,12 @@ def test_kahane_simplify1():
     t = (G(mu)*G(nu)*G(rho)*G(sigma)*G(-mu))
     r = kahane_simplify(t)
     assert r.equals(-2*G(sigma)*G(rho)*G(nu))
-    t = (G(mu)*G(nu)*G(rho)*G(sigma)*G(-mu))
+    t = (G(mu)*G(-mu)*G(rho)*G(sigma))
     r = kahane_simplify(t)
-    assert r.equals(-2*G(sigma)*G(rho)*G(nu))
-
+    assert r.equals(4*G(rho)*G(sigma))
+    t = (G(rho)*G(sigma)*G(mu)*G(-mu))
+    r = kahane_simplify(t)
+    assert r.equals(4*G(rho)*G(sigma))
```

Comment:
Again actually is equals needed here?

Reply:
(Rather than `==`)

Reply:
It doesn't need to be equals; `==` works fine. In fact it would even work to replace all of the `.equals` with `==` in this function (`test_kahane_simplify1()`). Would that be preferable, or should I just fix it in the two new assert statements?

Reply:
Oh, I hadn't really looked at the rest of the code. In general `.equals` which calls `simplify` can be slow and should not be used in test code. It also does not assert the precise form of the output since its purpose is to determine that two possibly different expressions are equivalent rather than being precisely the same expression. Maybe that's deliberate or maybe not.

For now though we can leave that unchanged. You've done the right thing by making the new tests use the same style as the existing ones.